### PR TITLE
opencv: Overwrite PV so that it isn't too long

### PR DESCRIPTION
**Pull request by chaitu236**
PV of opencv from openembedded-core contains git hashes of [5 repos](https://github.com/ni/meta-openembedded/blob/nilrt/master/sumo/meta-oe/recipes-support/opencv/opencv_3.3.bb#:~:text=SRCREV_opencv%20%3D%20%2287c27a074db9f6d9d60513f351daa903606ca370%22,SRCREV_vgg%20%3D%20%22fccf7cd6a4b12079f73bbfb21745f9babcd4eb1d%22).
This causes package name and thereby the export path to be too long
causing build failure in nibuild. Overwriting PV here to avoid this.

### Testing
Verified locally built ipks don't have git hashes anymore

### Note
If we are able to fix the max path length at source, will revert this change.
This is a hot fix to get CI passing.

### CODE REVIEW

**Comment by chaitu236** [comment]
@ni/rtos

**Review by jeminor — APPROVED** [review]
I'm fine with this for now, but I'll defer to the maintainers.

**Comment by jeminor** [comment]
Might also make sense to just drop this in meta-openembedded since it doesn't appear to be done this way in hardknott at least:
https://github.com/ni/meta-openembedded/blob/nilrt/master/hardknott/meta-oe/recipes-support/opencv/opencv_4.5.2.bb

**Comment by chaitu236** [comment]
> Might also make sense to just drop this in meta-openembedded since it doesn't appear to be done this way in hardknott at least: https://github.com/ni/meta-openembedded/blob/nilrt/master/hardknott/meta-oe/recipes-support/opencv/opencv_4.5.2.bb

Good point. I'll do that

**Comment by chaitu236** [comment]
https://github.com/ni/meta-openembedded/pull/13

**Pull request closed without merging**
